services/black/src/api/DatabaseModel/GuestInformationRecord.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using artiso.AdsdHotel.Black.Contracts;
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace artiso.AdsdHotel.Black.Api.DatabaseModel
+{
+    class GuestInformationRecord
+    {
+        [BsonId]
```

Comment:
if we use the annotation here is our own gernerated GUID getting used or is it being replaced by a mongodb generated one?

Reply:
I think this is just a hint for mongodb to use it as an id.